2.8/debian-10/rootfs/opt/bitnami/scripts/kafka/setup.sh

```diff
@@ -43,5 +43,9 @@ for dir in "$KAFKA_LOG_DIR" "$KAFKA_CONF_DIR" "$KAFKA_MOUNTED_CONF_DIR" "$KAFKA_
 done
 # Ensure Kafka is initialized
 kafka_initialize
+# If KRaft is enabled initialize
+if is_boolean_yes "${KAFKA_ENABLE_KRAFT:-}"; then
```

Comment:
Same than the above:

```suggestion
if is_boolean_yes "$KAFKA_ENABLE_KRAFT"; then
```